### ISSUE TITLE
Extension only downloads workout for strength training 

### ISSUE THREAD

**Issue by Memo4ninja** [labsansis/garmin-workout-downloader#14]
Hi! Other types of activities (cardio, yoga, pilates etc.) can also have exercises but the extension only picks it up for strength training. Maybe which type of activities to get those details from could be an option?

**Comment by labsansis** [comment]
Thanks for reporting this here! The focus of this extension is certainly strength training but I'll look into this as well.

In what scenarios does a cardio workout have exercises? Do you have to load a pre-planned workout for this?

**Comment by Memo4ninja** [comment]
Yes pre-planned cardio workout use the same exercise data than the strength training. I don't know if that's a feature many people use? Personally I like to separate pure gym/strength workouts from cardio-focused ones but I still want to track both :)

**Comment by labsansis** [comment]
This took a long while with the holiday period and what not, but finally got around to it.

I removed the strength training filter which in my testing so far seems to download other workout type exercises as well. One thing to note is that the exercise data only seems contain basic info what happened, not what was planned. In other words, information from the structured workout such as exercise target types, number of circuits is not included. This is true for the Garnin Connect UI as well from what I can tell.

I've posted this change to both extension stores and they seem to be live already.
